backend/src/install/cleanup.rs

```diff
@@ -148,6 +148,28 @@ pub async fn cleanup(ctx: &RpcContext, id: &PackageId, version: &Version) -> Res
             .await
             .apply(|res| errors.handle(res));
     }
+    let assets_path = ctx
+        .datadir
+        .join(PKG_VOLUME_DIR)
+        .join(id)
+        .join("assets")
+        .join(version.as_str());
```

Comment:
I think there's a function `assets_dir` in volumes.rs that would be better to use for this